### PR TITLE
Simple changes

### PR DESCRIPTION
**Pull request by bpradipt**
Minor changes to fix typos and missing manifest file

**- Description for the changelog**

Minor changes to fix typos and missing manifest file

### CODE REVIEW

File: docs/DEVELOPMENT.md [hunk 1]
```diff
@@ -20,9 +20,15 @@ In summary:
 
 
 ## Set Environment Variables
+
+Set your github userid
+```
+export GITHUB_USER=<user>
```
Comment (jensfr) [inline docs/DEVELOPMENT.md:26]: why not quay.io user id?

Comment (bpradipt) [reply docs/DEVELOPMENT.md:26]: yeah good suggest..

Comment (bpradipt) [reply docs/DEVELOPMENT.md]: done

**Review by pmores — APPROVED** [review]

**Review by jensfr — APPROVED** [review]
lgtm, thank you @bpradipt

**Pull request merged**